### PR TITLE
added remainder of 6 pin and 10 pin tag connects. Made schematic symb…

### PR DESCRIPTION
**Pull request by electricfungi**
added remainder of 6 pin and 10 pin tag connects. Made schematic symbol and land patterns match datasheets. Added supports at component level.

### CODE REVIEW

File: components/tag-connect/TC2030-IDC-NL.stanza [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  ; landing pads
+  for (l in grid-locs(2, 3, 1.27, 1.27), i in [2 4 6 1 3 5]) do :
+    pad p[i] : testpoint-pad(0.787) at l
+  
+  ; alignment holes
+  layer(Cutout()) = Circle((- 2.54), 0.0, 0.4955)
+  layer(Cutout()) = Circle(2.54, 1.016, 0.4955)
+  layer(Cutout()) = Circle(2.54, (- 1.016), 0.4955)
+
+  layer(Courtyard(Top)) = Rectangle(6.08, 3.032 ) ;, loc((-0.692), 0.0))
```
Comment (bpellenard) [inline components/tag-connect/TC2030-IDC-NL.stanza:30]: Do you want to use the loc in the rectangle? If not, please delete it

File: components/tag-connect/TC2050-IDC-NL.stanza [hunk 2]
```diff
@@ -12,26 +12,60 @@ defpackage ocdb/tag-connect/TC2050-IDC-NL :
...
+    swd([SWD-SWO, SWD-TRACESWO]).swo => self.p[6]
+    swd([SWD-SWO, SWD-TRACESWO]).traceswo => self.p[8]
+
+  supports power:
+    power.vdd => self.p[1]
+    power.gnd => self.p[3]
+    ; power.gnd => self.p[5]
+    ; power.gnd => self.p[7]
+    ; power.gnd => self.p[9]
+    ; net (power.gnd self.p[5])
```
Comment (bpellenard) [inline components/tag-connect/TC2050-IDC-NL.stanza:67]: Do you want to keep these lines?

Comment (electricfungi) [reply components/tag-connect/TC2050-IDC-NL.stanza:67]: yes, current;ly we can't do this. I would like this feature added. This also gives a heads up to folks using this to connect these pins IF they use the arm jtag

File: components/tag-connect/TC2030-IDC-NL.stanza (hunk 1 above)
Comment (electricfungi) [reply components/tag-connect/TC2030-IDC-NL.stanza]: deleted

**Review by bpellenard — APPROVED** [review]
Looks good to me!

**Pull request merged**